gcloud/pubsub/_gax.py

```diff
@@ -180,7 +181,279 @@ def topic_list_subscriptions(self, topic_path):
         return subs, response.next_page_token
 
 
+class _SubscriberAPI(object):
+    """Helper mapping subscriber-related APIs.
+
+    :type gax_api: :class:`google.pubsub.v1.publisher_api.SubscriberApi`
+    :param gax_api: API object used to make GAX requests.
+    """
+    def __init__(self, gax_api):
+        self._gax_api = gax_api
+
+    def list_subscriptions(self, project):
+        """List subscriptions for the project associated with this API.
+
+        See:
+        https://cloud.google.com/pubsub/reference/rest/v1/projects.subscriptions/list
```

Comment:
Deep links to the specific API endpoints are [broken](https://github.com/googleapis/googleapis/issues/31).  We can only point to the [Subscriber API](https://cloud.google.com/pubsub/reference/rpc/google.pubsub.v1#google.pubsub.v1.Subscriber).